packages/nox/test/tpetra/ME_Tpetra_Heq_def.hpp

```diff
@@ -276,11 +274,10 @@ evalModelImpl(const Thyra::ModelEvaluatorBase::InArgs<Scalar> &inArgs,
 
       // Reduce local contributions and fill the vector
       if (comm_->getSize() > 1) {
-        localSum->template modify<execution_space>();
         localSum->reduce();
         if (myRank == proc) {
-          view_type source = localSum->template getLocalView<execution_space>();
-          view_type target = integralOp_->template getLocalView<execution_space>();
+          auto source = localSum->template getLocalView<execution_space>(Tpetra::Access::ReadOnly);
+          auto target = integralOp_->template getLocalView<execution_space>(Tpetra::Access::ReadWrite);
```

Comment:
Should these be changed to getLocalViewDevice(...) instead of using the template parameter?

Reply:
Fixed